### PR TITLE
[Bug 14889] com.livecode.math: Emit errors when converting to base 10.

### PR DESCRIPTION
**Pull request by peter-b**

### CODE REVIEW

File: libscript/src/module-math.cpp [hunk 1]
```diff
@@ -369,6 +369,10 @@ extern "C" MC_DLLEXPORT void MCMathEvalConvertToBase10(MCStringRef p_operand, in
         else
             r_output = t_negative ? -t_result : t_result;
     }
+	else if (t_error)
```
Comment (runrevmark) [inline libscript/src/module-math.cpp:372]: This isn't quite right... If MCMathConvertToBase10 returns false then it means it threw an error (failure to allocate memory)... If it returns 'true' then 't_error' will indicate whether it managed to convert or not.

Comment (runrevmark) [inline libscript/src/module-math.cpp:372]: Sorry - just noticed the init of 't_error' :)

**Comment by peter-b** [comment]
Requires a release note.

**Comment by runrevmark** [comment]
Hmmm - that's a good point - are we doing release notes in the current scheme for LCB-only related things?

Perhaps we should consider this slightly... We don't want LCB things to be lost amongst LCS things, my feeling is it should be a separate thing somehow (or at least in a separate section).

**Comment by peter-b** [comment]
@runrevmark Yes, I think it would be a good idea to separate things out.  Something to discuss on Monday?

**Comment by livecodefraser** [comment]
Reviewed-By: Fraser Gordon

**Pull request merged**